index.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var _               = require("lodash");
```

Comment:
You aren't using a lot of lodash so to save the end user having to download it all on install, we could depend on the individual methods instead (which are independently published on npm). If you want `_.isString` for example, we could depend on [lodash.isstring](https://www.npmjs.com/package/lodash.isstring) and `require('lodash.isstring')` here.